assets/scss/base/form/_form.scss

```diff
@@ -399,22 +399,21 @@ Styleguide Form.control
 */
 
 .form-control {
+  @include box-sizing(border-box);
   @include core-19;
-
   width: 100%;
-  @include box-sizing(border-box);
-  padding: 4px;
+
+  padding: 5px 4px 4px;
   background-color: $white;
-  border: 1px solid $border-colour;
+  border: 2px solid $text-colour;
 
-  // Disable appearance and remove rounded corners
-  appearance: none;
+  // Disable webkit appearance and remove rounded corners
+  -webkit-appearance: none;
```

Comment:
`-webkit-appearance` fails stylelint: https://travis-ci.org/hmrc/assets-frontend/builds/283585835#L1555.

Autoprefixer takes care of our vendor prefixes for us 😄

Reply:
Updated and removed from other _form.scss too. 👍